### ISSUE TITLE
Error opening output after it has been closed once

### ISSUE THREAD

**Issue by drj-io** [justinlatimer/node-midi#124]
This is on a 2011 MBP with Sierra & Yosemite 

We're using the `easymidi` wrapper for this.  Unable to get around this error after sending midi data on the second pass.  

```
MidiOutCore::initialize: error creating OS-X MIDI client object.

libc++abi.dylib: terminating with uncaught exception of type RtMidiError
Abort trap: 6
```

Can't seem to reproduce on a _newer_ mac computer with Sierra. Would be great if we could catch this, through try/catch or an callback with error.

**Comment by drj-io** [comment]
```javascript
var output = new easymidi.Output(moduleSettings.output_midi_device);
output.send('cc', opts)
output.close();
```

This fails on the second pass.

**Comment by justinlatimer** [comment]
Updated RtMidi in 7c659437963c311bc62d94898ad8867e220a8899, published to npm in `v2.0.0`
